From fe01c9ec4979c78b6a7caf045cab17ba52d4ad08 Mon Sep 17 00:00:00 2001
From: Brad Barden <brad+github@mifflinet.net>
Date: Sun, 11 Jan 2015 17:52:11 -0600
Subject: [PATCH] bugfix header in platform/redhat/rsyslog.conf

the "For more information" doc path includes the string "*/", breaking c-style comment format. fixes #202.
---
 platform/redhat/rsyslog.conf | 19 +++++++++----------
 1 file changed, 9 insertions(+), 10 deletions(-)

diff --git a/platform/redhat/rsyslog.conf b/platform/redhat/rsyslog.conf
index 340874d..3df6c7e 100644
--- a/platform/redhat/rsyslog.conf
+++ b/platform/redhat/rsyslog.conf
@@ -1,13 +1,12 @@
-/* rsyslog configuration file (for Red Hat-based systems)
- * note that most of this config file uses old-style format,
- * because it is well-known AND quite suitable for simple cases
- * like we have with the default config. For more advanced 
- * things, RainerScript configuration is suggested.
- *
- * For more information see /usr/share/doc/rsyslog-*/rsyslog_conf.html
- * or latest version online at http://www.rsyslog.com/doc/rsyslog_conf.html 
- * If you experience problems, see http://www.rsyslog.com/doc/troubleshoot.html
- */
+# rsyslog configuration file (for Red Hat-based systems)
+# note that most of this config file uses old-style format,
+# because it is well-known AND quite suitable for simple cases
+# like we have with the default config. For more advanced 
+# things, RainerScript configuration is suggested.
+#
+# For more information see /usr/share/doc/rsyslog-*/rsyslog_conf.html
+# or latest version online at http://www.rsyslog.com/doc/rsyslog_conf.html 
+# If you experience problems, see http://www.rsyslog.com/doc/troubleshoot.html
 
 #### MODULES ####
 
